src/coreComponents/mesh/generators/CellBlockManagerABC.hpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2020-     GEOSX Contributors
+ * All right reservepd
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+#ifndef GEOSX_CELLBLOCKMANAGERABC_HPP
+#define GEOSX_CELLBLOCKMANAGERABC_HPP
+
+#include "dataRepository/Group.hpp"
+
+#include <map>
+
+namespace geosx
+{
+
+/**
+ * @brief Abstract base class for CellBlockManager.
```

Comment:
I think we need to have more details here to what is a `CellBlockManager` in GEOSX

Reply:
Absolutely.